lib/modules/manager/gitlabci/utils.ts

```diff
@@ -24,14 +24,32 @@ const depProxyRe = regEx(
  * @param imageName as used in .gitlab-ci.yml file
  * @return package dependency for the image
  */
-export function getGitlabDep(imageName: string): PackageDependency {
+export function getGitlabDep(
+  imageName: string,
+  registryAliases: Record<string, string> = {}
+): PackageDependency {
   const match = depProxyRe.exec(imageName);
   if (match?.groups) {
     const dep = { ...getDep(match.groups.depName), replaceString: imageName };
     // TODO: #7154
     dep.autoReplaceStringTemplate = `${match.groups.prefix}${dep.autoReplaceStringTemplate}`;
     return dep;
-  } else {
-    return getDep(imageName);
   }
+
+  for (const [name, value] of Object.entries(registryAliases)) {
```

Comment:
```suggestion
  for (const [name, value] of Object.entries(registryAliases ?? {})) {
```
better set default here otherwise passing `null` at runtime will cause an error. default value on function parameter only works for `undefined`